ModelFiles/xml/yeastGEM.xml

```diff
@@ -199653,12 +199653,17 @@
         <notes>
           <body xmlns="http://www.w3.org/1999/xhtml">
             <p>Confidence Level: 0</p>
-            <p>NOTES: added after Gapfilling (PR #185); MNXR94845</p>
+            <p>NOTES: added after Gapfilling (PR #185); seed:rxn09802; unknown transporter</p>
```

Comment:
SEED ids are also able to be stored in annotation, use the `rxnSEEDID` field for consistency (notes should only store what cannot go in annotation).

Reply:
If we are going to include seed IDs in the model, should we include them all for all possible reactions?

Reply:
I have opened a new issue #231  to add rhea and modelseed IDs in a systematic way. So here, we don't address this.